src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx

```diff
@@ -135,13 +125,7 @@ export const ProfileIntro = ({
           loading={loading}
           avatarSize="large"
           usernameAs={usernameAs}
-        >
-          {contributor && !!contributor.website && (
-            <ExternalLink href={contributor.website}>
-              <LinkIcon /> {contributor.website}
-            </ExternalLink>
-          )}
-        </ContributorLarge>
+        ></ContributorLarge>
```

Comment:
A simplifier pour :
``` js
<ContributorLarge
          contributor={contributor}
          onSubscribe={subscribe}
          onUnsubscribe={unsubscribe}
          loading={loading}
          avatarSize="large"
          usernameAs={usernameAs}
        />
```

Reply:
that's why webstorm was highlighting it =D